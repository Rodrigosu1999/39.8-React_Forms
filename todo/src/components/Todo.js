import React, { useState } from "react";
import defaultTodoList from "../helpers/defaultTodoList"

function Todo({id, task, deleteTodo}) {
    return (
        <div className="Todo"  id={id}>
            <span className="Todo-task">
                {task}
            </span>
            <button onClick={deleteTodo}>X</button>
        </div>
    );
}

export default Todo;