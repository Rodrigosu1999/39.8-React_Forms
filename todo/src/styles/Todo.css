.Todo{
    margin: 5px;
}

.Todo-task{
    font-size: 1.5em;
}