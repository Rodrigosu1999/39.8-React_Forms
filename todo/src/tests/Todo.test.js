import React from 'react';
import { getByText, render, screen } from '@testing-library/react';
import Todo from '../components/Todo';

it('Renders App', () => {
  render(<Todo />);
});

it('Passes Snapshot', () => {
  const {asFragment} = render(<Todo />);
  expect(asFragment()).toMatchSnapshot();
});

it('renders correct text', () => {
  const {getByText} = render(<Todo />);
  console.log(getByText('X'))
});