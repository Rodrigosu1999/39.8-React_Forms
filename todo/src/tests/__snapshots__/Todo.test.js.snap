// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Passes Snapshot 1`] = `
<DocumentFragment>
  <div
    class="Todo"
  >
    <span
      class="Todo-task"
    />
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
